internal/contour/translator.go

```diff
@@ -247,26 +243,6 @@ func (t *Translator) removeSecret(s *v1.Secret) {
 	t.recomputeTLSListener(t.cache.ingresses, t.cache.secrets)
 }
 
-// writeSecret writes the contents of the secret to a fixed location on
-// disk so that envoy can pick them up.
-// TODO(dfc) this is due to https://github.com/envoyproxy/envoy/issues/1357
-func (t *Translator) writeCerts(s *v1.Secret) {
```

Comment:
i'm super grad we can get rid of this crap

Reply:
Yeah if felt good!